### ISSUE TITLE
Add support for deploying AWS Lambda Layers

### ISSUE THREAD

**Issue by Moke** [aws/aws-toolkit-azure-devops#356]
It would appear the ability to publish AWS layers from azure devops pipelines is not currently possible.

I would like to request that the ability to publish layers similarly to the existing aws lambda function deployment.

**Comment by IOnlyFetchBranches** [comment]
I'd like to necro this really quick, because i don't understand why we *don't* have this functionality already?

**Comment by bryceitoc9** [comment]
Sorry about the delay, taking a look at this, I'm assuming the request here is to create a layer from a prebuilt zip or directory on the pipeline agent or from S3 (as opposed to building the layer, e.g. `npm install`)?

**Comment by IOnlyFetchBranches** [comment]
yes, I do believe so. that would be sick functionality to have in the
toolkit :)

**Comment by Moke** [comment]
> Sorry about the delay, taking a look at this, I'm assuming the request here is to create a layer from a prebuilt zip or directory on the pipeline agent or from S3 (as opposed to building the layer, e.g. `npm install`)?


Any ability at all to publish a layer would be helpful. Today i generate a zip file and manually copy it into the aws console.  It's rare i make changes, but a PITA to update it.

**Comment by ajfis** [comment]
A possible workaround is to use the AWS CLI to deploy the layers
